crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -1808,15 +1835,40 @@ impl<'a> EntityCommands<'a> {
     /// # bevy_ecs::system::assert_is_system(example_system);
     pub fn clone_and_spawn_with(
         &mut self,
-        f: impl FnOnce(&mut EntityCloneBuilder) + Send + Sync + 'static,
+        config: impl FnOnce(&mut EntityCloneBuilder) + Send + Sync + 'static,
     ) -> EntityCommands<'_> {
         let entity_clone = self.commands().spawn_empty().id();
-        self.queue(clone_and_spawn_with(entity_clone, f));
+        self.queue(clone_with(entity_clone, config));
         EntityCommands {
             commands: self.commands_mut().reborrow(),
             entity: entity_clone,
         }
     }
+
+    /// Clones the specified components of this entity and inserts them into another entity.
+    ///
+    /// Components can only be cloned if they implement
+    /// [`Clone`] or [`Reflect`](bevy_reflect::Reflect).
+    ///
+    /// # Panics
+    ///
+    /// The command will panic when applied if either of the entities do not exist.
```

Comment:
Gah more panics :( Not your problem: this needs a holistic solution.

Reply:
I do have another PR for that :P (not holistic, just commands). Not sure how it'll interact with work in the fallible systems area though